cmd/tiller-proxy/main.go

```diff
@@ -135,7 +130,17 @@ func main() {
 	}
 
 	proxy = tillerProxy.NewProxy(kubeClient, helmClient, timeout)
-	chartClient := chartUtils.NewChartClient(kubeClient, appRepoClient, userAgent())
+	kubeappsNamespace := os.Getenv("POD_NAMESPACE")
+	if kubeappsNamespace == "" {
+		log.Fatalf("POD_NAMESPACE should be defined")
+	}
```

Comment:
Much better, thanks for moving it out to main.go even though it wasn't a change you'd added.